source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -447,6 +460,38 @@ TEST_F(NiDCPowerDriverApiTest, ConfigureOutputFunctionAndCurrentLevel_Configures
   EXPECT_EQ(expected_output_function_value, actual_output_function_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, FetchMultiple_ReturnsResultOfExpectedSize)
```

Comment:
Can give part of test name cover the steps of set attribute and initiate?

Reply:
Can test name be 
MeasurementIsInitiated_FetchMulitple_ReturnsResultOfExpectedSize

Reply:
Yep, just updated to `SetMeasureWhenAndInitiate_FetchMultiple_ReturnsResultOfExpectedSize`